addons/pos_restaurant/static/src/app/services/pos_store.js

```diff
@@ -46,6 +48,54 @@ patch(PosStore.prototype, {
         }
         return super.defaultScreen;
     },
+    createNewOrder(data) {
+        const order = super.createNewOrder(data);
+
+        if (order.table_id) {
+            order.setCustomerCount(order.table_id.seats);
```

Comment:
```suggestion
            order.setCustomerCount();
```

`setCustomerCount` should expect an order object, not a number.

Reply:
wait, `setCustomerCount` is defined in `pos_store`, not in `pos_order`. This line of code can't be working?